tools/preflight/preflight.sh

```diff
@@ -150,77 +156,77 @@ check_if_ocp() {
   if [[ $(kubectl api-resources | grep openshift.io) ]]; then
     is_ocp="Y"
   fi
-  echo "${is_ocp}"
+  echolog "${is_ocp}"
 }
 
 check_helm_version() {
-  echo -e "${LIGHT_BLUE}Checking for required Helm version (>= v${MIN_HELM_VERSION})...${NC}\n"
+  echolog -e "${LIGHT_BLUE}Checking for required Helm version (>= v${MIN_HELM_VERSION})...${NC}\n"
   local exit_status=0
 
   # Abort successfully in case of OCP setup
   if [[ $(check_if_ocp) == "Y" ]]; then
-    echo -e "${GREEN} ${CHECK} Helm not needed for OCP clusters${NC}\n"
+    echolog -e "${GREEN} ${CHECK} Helm not needed for OCP clusters${NC}\n"
     return ${exit_status}
   fi
 
-  if ! command -v "helm" >/dev/null 2>&1; then
-    echo -e "${RED} ${CROSS} Unable to find helm${NC}\n"
+  if ! command -v "helm" &>> preflight-log-${TIMESTAMP}.out; then
```

Comment:
create variable this log file name and reuse

Reply:
resolved